Dockerfile.builder

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
```

Comment:
I guess it would be simpler to publish the image on Docker Hub once for all, instead (but we'd need to keep this `dockerfile` somewhere, though..)